### ISSUE TITLE
`enforce-consistent-line-wrapping` leaving line too long

### ISSUE THREAD

**Issue by aguo-ntech** [schoero/eslint-plugin-readable-tailwind#366]
### Description

The line remains longer than the `printWidth`; I expect this to never happen when using the rule.

### Flavor

JSX

### Code Input

```tsx
console.log(
	<div>
		<div>
			<div>
				<div>
					<div>
						<div>
							<div
								className="
									translate-y-0 opacity-100 transition-all duration-200 ease-out starting:translate-y-1 starting:opacity-0
								"
							/>
						</div>
					</div>
				</div>
			</div>
		</div>
	</div>
);
```

### Expected Behavior

```tsx
The classes would be split up into multiple lines, since the line is longer than the `printWidth`.
```

### Actual Behavior

```tsx
No reports from the rule.
```

### Reproduction URL

https://github.com/abrahamguo/repro/tree/class-wrapping

### ESLint Log

```shell
repro/index.jsx
  10:1  error  This line has a length of 140. Maximum allowed is 120  max-len

✖ 1 problem (1 error, 0 warnings)
```

### ESLint Config

```typescript
import eslintPluginBetterTailwindcss from 'eslint-plugin-better-tailwindcss';
import parser from '@typescript-eslint/parser';

export default {
	files: [`index.jsx`],
	languageOptions: { parser },
	plugins: { 'better-tailwindcss': eslintPluginBetterTailwindcss },
	rules: {
		'max-len': [`error`, { code: 120 }],
		'better-tailwindcss/enforce-consistent-line-wrapping': [
			`error`,
			{ printWidth: 120, group: `never`, indent: `tab` }
		]
	}
};
```

### Versions

- ESLint: 10.2.0
- Parser: `@typescript-eslint/parser`@8.58.2
- Plugin: 4.4.1
- Node: 25.7.0

**Comment by schoero** [comment]
The issue was that the `max-len` rule counts the tab width differently than eslint-plugin-better-tailwindcss.

There is now a new [`tabWidth` property](https://github.com/schoero/eslint-plugin-better-tailwindcss/blob/main/docs/rules/enforce-consistent-line-wrapping.md#tabwidth). You can set it to `2` to solve your issue.
